.travis.yml

```diff
@@ -22,6 +22,11 @@ notifications:
     on_success: never
     on_failure: always
 
+before_script:
+  - 'if [ "$IMMUTABLE_VERSION" ]; then npm install immutable@$IMMUTABLE_VERSION; fi'
+env:
+  - IMMUTABLE_VERSION=3
+  - IMMUTABLE_VERSION=4.0.0-rc.12
```

Comment:
Can this accept a broader version, such as `IMMUTABLE_VERSION=4` or something?

Reply:
only after release, this version does not follow semver :(

Reply:
Yeah I realized that, that's why I went with `^4.0.0-rc`. When 4 gets stable, I'll revise that.